2025-07-23T16:01:40.975877089Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/base.py", line 236, in run
2025-07-23T16:01:40.975880659Z     super().run()
2025-07-23T16:01:40.975884259Z     ~~~~~~~~~~~^^
2025-07-23T16:01:40.975888099Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/base.py", line 72, in run
2025-07-23T16:01:40.975891689Z     Arbiter(self).run()
2025-07-23T16:01:40.975895339Z     ~~~~~~~^^^^^^
2025-07-23T16:01:40.975898919Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/arbiter.py", line 58, in __init__
2025-07-23T16:01:40.97590276Z     self.setup(app)
2025-07-23T16:01:40.9759068Z     ~~~~~~~~~~^^^^^
2025-07-23T16:01:40.97591068Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/arbiter.py", line 118, in setup
2025-07-23T16:01:40.97591435Z     self.app.wsgi()
2025-07-23T16:01:40.97591792Z     ~~~~~~~~~~~~~^^
2025-07-23T16:01:40.97592168Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/base.py", line 67, in wsgi
2025-07-23T16:01:40.97592579Z     self.callable = self.load()
2025-07-23T16:01:40.97592949Z                     ~~~~~~~~~^^
2025-07-23T16:01:40.975936491Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2025-07-23T16:01:40.975940351Z     return self.load_wsgiapp()
2025-07-23T16:01:40.975944161Z            ~~~~~~~~~~~~~~~~~^^
2025-07-23T16:01:40.975948401Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2025-07-23T16:01:40.975952931Z     return util.import_app(self.app_uri)
2025-07-23T16:01:40.975956581Z            ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
2025-07-23T16:01:40.975960321Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/util.py", line 371, in import_app
2025-07-23T16:01:40.975963971Z     mod = importlib.import_module(module)
2025-07-23T16:01:40.975973921Z   File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
2025-07-23T16:01:40.975978321Z     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-23T16:01:40.975982272Z            ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-23T16:01:40.975986222Z   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
2025-07-23T16:01:40.975990062Z   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
2025-07-23T16:01:40.975993732Z   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
2025-07-23T16:01:40.976010372Z   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
2025-07-23T16:01:40.976012972Z   File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
2025-07-23T16:01:40.976015232Z   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
2025-07-23T16:01:40.976017342Z   File "/opt/render/project/src/backend/config/wsgi.py", line 16, in <module>
2025-07-23T16:01:40.976019473Z     application = get_wsgi_application()
2025-07-23T16:01:40.976021842Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
2025-07-23T16:01:40.976024002Z     django.setup(set_prefix=False)
2025-07-23T16:01:40.976026113Z     ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
2025-07-23T16:01:40.976028503Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/django/__init__.py", line 24, in setup
2025-07-23T16:01:40.976030623Z     apps.populate(settings.INSTALLED_APPS)
2025-07-23T16:01:40.976032713Z     ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-23T16:01:40.976034983Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/django/apps/registry.py", line 124, in populate
2025-07-23T16:01:40.976037153Z     app_config.ready()
2025-07-23T16:01:40.976039353Z     ~~~~~~~~~~~~~~~~^^
2025-07-23T16:01:40.976041523Z   File "/opt/render/project/src/backend/ai_service/apps.py", line 11, in ready
2025-07-23T16:01:40.976044063Z     from . import signals
2025-07-23T16:01:40.976046283Z   File "/opt/render/project/src/backend/ai_service/signals.py", line 7, in <module>
2025-07-23T16:01:40.976048523Z     from ai_services.common.signals import ai_usage_logged
2025-07-23T16:01:40.976050573Z ModuleNotFoundError: No module named 'ai_services'
2025-07-23T16:01:42.796743945Z ==> Exited with status 1
2025-07-23T16:01:42.814268543Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys
2025-07-23T16:01:49.786389576Z ==> Running '     gunicorn config.wsgi:client-nest'
2025-07-23T16:01:57.872872441Z /opt/render/project/src/.venv/lib/python3.13/site-packages/drf_yasg/__init__.py:2: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
2025-07-23T16:01:57.872891162Z   from pkg_resources import DistributionNotFound, get_distribution
2025-07-23T16:01:59.435064674Z Traceback (most recent call last):
2025-07-23T16:01:59.436671943Z   File "/opt/render/project/src/.venv/bin/gunicorn", line 8, in <module>
2025-07-23T16:01:59.436684813Z     sys.exit(run())
2025-07-23T16:01:59.436689273Z              ~~~^^
2025-07-23T16:01:59.436693123Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/wsgiapp.py", line 67, in run
2025-07-23T16:01:59.436697094Z     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
2025-07-23T16:01:59.436700594Z     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
2025-07-23T16:01:59.436704094Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/base.py", line 236, in run
2025-07-23T16:01:59.436707514Z     super().run()
2025-07-23T16:01:59.436710854Z     ~~~~~~~~~~~^^
2025-07-23T16:01:59.436715734Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/base.py", line 72, in run
2025-07-23T16:01:59.436721644Z     Arbiter(self).run()
2025-07-23T16:01:59.436727154Z     ~~~~~~~^^^^^^
2025-07-23T16:01:59.436732484Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/arbiter.py", line 58, in __init__
2025-07-23T16:01:59.436737884Z     self.setup(app)
2025-07-23T16:01:59.436743204Z     ~~~~~~~~~~^^^^^
2025-07-23T16:01:59.436748635Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/arbiter.py", line 118, in setup
2025-07-23T16:01:59.436753655Z     self.app.wsgi()
2025-07-23T16:01:59.436756985Z     ~~~~~~~~~~~~~^^
2025-07-23T16:01:59.436760375Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/base.py", line 67, in wsgi
2025-07-23T16:01:59.436766095Z     self.callable = self.load()
2025-07-23T16:01:59.436771805Z                     ~~~~~~~~~^^
2025-07-23T16:01:59.436777085Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2025-07-23T16:01:59.436781996Z     return self.load_wsgiapp()
2025-07-23T16:01:59.436787556Z            ~~~~~~~~~~~~~~~~~^^
2025-07-23T16:01:59.436792926Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2025-07-23T16:01:59.436796796Z     return util.import_app(self.app_uri)
2025-07-23T16:01:59.436800136Z            ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
2025-07-23T16:01:59.436803446Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/gunicorn/util.py", line 371, in import_app
2025-07-23T16:01:59.436806816Z     mod = importlib.import_module(module)
2025-07-23T16:01:59.436823286Z   File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
2025-07-23T16:01:59.436829227Z     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-23T16:01:59.436834587Z            ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-23T16:01:59.436839677Z   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
2025-07-23T16:01:59.436844787Z   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
2025-07-23T16:01:59.436849927Z   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
2025-07-23T16:01:59.436866478Z   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
2025-07-23T16:01:59.436868898Z   File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
2025-07-23T16:01:59.436871008Z   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
2025-07-23T16:01:59.436873098Z   File "/opt/render/project/src/backend/config/wsgi.py", line 16, in <module>
2025-07-23T16:01:59.436875168Z     application = get_wsgi_application()
2025-07-23T16:01:59.436877318Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
2025-07-23T16:01:59.436879368Z     django.setup(set_prefix=False)
2025-07-23T16:01:59.436881398Z     ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
2025-07-23T16:01:59.436883508Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/django/__init__.py", line 24, in setup
2025-07-23T16:01:59.436885608Z     apps.populate(settings.INSTALLED_APPS)
2025-07-23T16:01:59.436887778Z     ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-23T16:01:59.436890438Z   File "/opt/render/project/src/.venv/lib/python3.13/site-packages/django/apps/registry.py", line 124, in populate
2025-07-23T16:01:59.436892598Z     app_config.ready()
2025-07-23T16:01:59.436894688Z     ~~~~~~~~~~~~~~~~^^
2025-07-23T16:01:59.436896928Z   File "/opt/render/project/src/backend/ai_service/apps.py", line 11, in ready
2025-07-23T16:01:59.436899028Z     from . import signals
2025-07-23T16:01:59.436901378Z   File "/opt/render/project/src/backend/ai_service/signals.py", line 7, in <module>
2025-07-23T16:01:59.436903548Z     from ai_services.common.signals import ai_usage_logged
2025-07-23T16:01:59.436905659Z ModuleNotFoundError: No module named 'ai_services'